apps/web/src/pages/integrations/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { IEnvironment, providers } from '@novu/shared';
+
+import { IntegrationEntity } from './IntegrationsStorePage';
+import { CONTEXT_PATH } from '../../config';
+import { CHANNEL_TYPE_TO_STRING } from '../../utils/channels';
+
+export const mapToTableIntegration = (integration: IntegrationEntity, environments?: IEnvironment[]) => {
+  const provider = providers.find((el) => el.id === integration.providerId);
+  const logoFileName = provider?.logoFileName
+    ? {
+        light: `${CONTEXT_PATH}/static/images/providers/light/${provider.logoFileName.light}`,
+        dark: `${CONTEXT_PATH}/static/images/providers/dark/${provider.logoFileName.dark}`,
+      }
+    : {
+        light: `${CONTEXT_PATH}/static/images/logo.png`,
+        dark: `${CONTEXT_PATH}/static/images/logo-light.png`,
+      };
```

Comment:
the provider rectangle logos are not delivered by the design yet, that's why it doesn't look ok on the screenshots